src/huggingface_hub/hf_api.py

```diff
@@ -405,6 +405,14 @@ def logout(self, token: Optional[str] = None) -> None:
         r = requests.post(path, headers={"authorization": "Bearer {}".format(token)})
         r.raise_for_status()
 
+    @staticmethod
+    def set_access_token(access_token: str):
+        write_to_credential_store("username", access_token)
```

Comment:
I feel like `hf_access_token` or the like would be more explicit than `username` for someone looking at their git credential manager from outside `huggingface_hub`, WDYT ?

Reply:
How about `hf_user` as a middle ground between @julien-c's proposal and yours?

Reply:
OK

I just don't want people to think that they need to replace it (it's just a dummy as it's not actually used)